capstone/src/main/java/com/google/sps/servlets/utils/ServletUtil.java

```diff
@@ -47,8 +47,6 @@ public static String getRedirectUri(HttpServletRequest req) {
     // If you want to run this locally, you will need to replace this with your dev server URI
     // - then add "/oauth2callback" to the end of it and add that to you API console under
     // Authorized URIs.
-    // return
-    // "https://8080-4417b0ad-e7ff-4d2f-acc8-c9ddcf7b56d9.us-west1.cloudshell.dev/oauth2callback";
     // return "https://clubhub-step-2020.googleplex.com/oauth2callback";
```

Comment:
nit: remove this too?

Reply:
I think it might be better to actually keep that return statement instead, since it is for the deployed site, so I will go ahead and remove the one I currently have and remove the comment for the deployed URL.